src/sentry/api/endpoints/team_projects.py

```diff
@@ -93,24 +100,35 @@ def get(self, request: Request, team) -> Response:
             paginator_cls=OffsetPaginator,
         )
 
+    @extend_schema(
+        # Ensure POST is in the projects tab
+        tags=["Projects"],
+        operation_id="Create a New Project",
+        parameters=[
+            GLOBAL_PARAMS.ORG_SLUG,
+            GLOBAL_PARAMS.TEAM_SLUG,
+            GLOBAL_PARAMS.NAME("The name of the project.", required=True),
+            GLOBAL_PARAMS.SLUG(
+                "Optional slug for the project. If not provided a slug is generated from the name."
+            ),
+            PROJECT_PARAMS.PLATFORM("The platform for the project."),
+            PROJECT_PARAMS.DEFAULT_RULES,
+        ],
+        request=ProjectPostSerializer,
+        responses={
+            201: sentry_project_serializer,
```

Comment:
nit: would change this name to `SentryProjectResponseSerializer` in the import rename above